sql/moz-fx-data-shared-prod/telemetry_derived/releases_v1/query.py

```diff
@@ -0,0 +1,42 @@
+# Import release data from https://product-details.mozilla.org/1.0/firefox.json
+
+from argparse import ArgumentParser
+from google.cloud import bigquery
+import io
+import json
+import requests
+from pathlib import Path
+import yaml
+
+URL = "https://product-details.mozilla.org/1.0/firefox.json"
+SCHEMA_FILE = Path(__file__).parent / "schema.yaml"
+
+parser = ArgumentParser(description=__doc__)
+parser.add_argument("--project", default="moz-fx-data-shared-prod")
+parser.add_argument("--destination_dataset", default="telemetry_derived")
+parser.add_argument("--destination_table", default="releases_v1")
+
+
+def main():
+    args = parser.parse_args()
+    client = bigquery.Client(args.project)
+
+    release_json = json.loads(requests.get(URL).text)
+    release_data = [r for _, r in release_json["releases"].items()]
```

Comment:
Is the key irrelevant? It appears to consistently be `<product>-<version>` so probably fine to drop. But would be worth adding a comment here about that decision.